src/addons/soc-generator/vivado/pins.csv

```diff
@@ -57,3 +57,29 @@ Comment,Name of the pin,“input” or “output”
 ,ja_7_o,input
 ,ja_7_i,output
 ,ja_7_oe,output
+,jd_0_o,input
+,jd_0_i,output
+,jd_0_oe,output
+,jd_1_o,input
+,jd_1_i,output
+,jd_1_oe,output
+,jd_2_o,input
+,jd_2_i,output
+,jd_2_oe,output
+,jd_3_o,input
+,jd_3_i,output
+,jd_3_oe,output
+,jd_4_o,input
+,jd_4_i,output
+,jd_4_oe,output
+,jd_5_o,input
+,jd_5_i,output
+,jd_5_oe,output
+,jd_6_o,input
+,jd_6_i,output
+,jd_6_oe,output
+,jd_7_o,input
+,jd_7_i,output
+,jd_7_oe,output
+,uart_rxd_out,output
```

Comment:
The UART pins are mono-direction pins that don't require a tristate IOBUF driver (like the LEDs, which also don't require a driver). But it can't hurt to have some, so I added them.